### DISCUSSION TITLE
List Languages is Empty

### DISCUSSION THREAD

**Discussion by robertbur** [requarks/wiki#3632] [Error / Bug Report]
OS: Debian 10
Wiki.js Version: Is the latest, 2.5.170
Database Engine: MariaDb

### Actual behaviour
I can't see the languages list.

### Steps to reproduce the behaviour
The other day I downloaded the Spanish Language for the wiki, and everything goes fine. However, today, when I tried to add another language, the list, is empty and I can only see the English and the Spanish Language.

Is any issue with the server or with my server?

I have the latest version of the Wiki, in a docker container on Debian.

Once again, thanks for your help.

Kind Regards

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)
